src/registry-client/src/cache-entry.ts

```diff
@@ -267,17 +264,8 @@ export class CacheEntry {
    */
   json(): JSONObj {
     if (this.#json !== undefined) return this.#json
-    const ser = serializedHeader && this.getHeader(serializedHeader)
-    if (ser) {
-      /* c8 ignore start - very rare, but theoretically possible to throw */
-      try {
-        return (this.#json = deserialize(ser) as JSONObj)
-      } catch {}
-      /* c8 ignore stop */
-    }
     const obj = JSON.parse(this.text()) as JSONObj
-    if (serializedHeader)
-      this.setHeader(serializedHeader, serialize(obj))
+    this.#json = obj
```

Comment:
This line is already handled in https://github.com/vltpkg/vltpkg/pull/314 but it's vital for the sake of running these benchmarks since non-v8-serialized-json objects were simply never getting memoized.